### ISSUE TITLE
minimum stability error

### ISSUE THREAD

**Issue by misaelnieto** [jmolivas/phpqa#2]
Hi.

Newbie here. I tried to install through composer on Fedora 22 and got an error:

```
  [InvalidArgumentException]                                                   
  Could not find package jmolivas/phpqa at any version for your minimum-stabi  
  lity (stable). Check the package spelling or your minimum-stability 
```

It's very likely that I lack some configuration, but I don't know where or what to look.

Attaching a screenshot of my console just because I can.

![nnieto mx-nnieto-lt _034](https://cloud.githubusercontent.com/assets/146899/8784142/4141c6e4-2ed5-11e5-8092-4f335e95bc9e.png)

**Comment by jmolivas** [comment]
@tzicatl may you try `dev-master`

```
$ composer global require jmolivas/phpqa:dev-master
```

**Comment by misaelnieto** [comment]
It's not getting better:
![nnieto mx-nnieto-lt _035](https://cloud.githubusercontent.com/assets/146899/8786137/6d15f0cc-2ee1-11e5-9e93-d878d7b46669.png)

**Comment by jmolivas** [comment]
@tzicatl: Can you please try instead, the `Cloning the project` instructions

https://github.com/jmolivas/phpqa#cloning-the-project

```
$ cd ~
$ git clone git@github.com:jmolivas/phpqa.git
$ cd phpqa
# download dependencies
$ composer install
# make phpqa globally accessible creating a symlink 
$ ln -s ~/phpqa/bin/phpqa.php /usr/local/bin/phpqa
```

Until I properly test the composer installaiton

**Comment by MontealegreLuis** [comment]
@tzicatl @jmolivas I tried it and I could install it with the following command.

```
$ composer global require jmolivas/phpqa:dev-master@dev
```

Although I found an issue installing it globally. It cannot find the binary files because it's a global installation and you would need to run

```
 parallel-lint
```

instead of

```
bin/parallel-lint
```

I will try to fix it if you're not already working on it.

**Comment by jmolivas** [comment]
Fixed [0.3.0](https://github.com/jmolivas/phpqa/releases/tag/0.3.0)
